### DISCUSSION TITLE
How can i get PX4' RCout data(actuator) on python API

### DISCUSSION THREAD

**Discussion by tuta4048** [microsoft/AirSim#3756] [Support Q&A]
## If you have question
* Add clear and concise title
* Add OS, AirSim version, Python version, Unreal version if applicable
* Include context on what you are trying to achieve
* Include details of what you already did to find answers

------------------------------------ (Remove above before filing the issue) ------------------------------------
*Ubuntu 16.4 / Airsim 4.22.4/ UE 4.22

i'm trying to enhance flight stabilization of multi-rotor by using reinforcement learning.

In real-world, many type of drones are vulnerable to disturbance.

To overcome this, i wanna try to change PID controllers of PX4 to RL algorithm.

Now, i'm testing the drone in HILS and  i want to put the all the actuator data as output variable, separately.
 
However, I don't know how to get the output of PX4's RCout.

How can i get this value? and is there any other way? 

I use Python API, Pixhawk 2.4.8 with PX4/audupilot code

**Comment by jonyMarino** [comment]
This should be now achievable through #3242
